adapters/criteo/criteo.go

```diff
@@ -67,7 +76,7 @@ func (a *adapter) MakeBids(request *openrtb2.BidRequest, requestData *adapters.R
 
 	for _, seatBid := range response.SeatBid {
 		for i := range seatBid.Bid {
-			var bidExt openrtb_ext.ExtBid
```

Comment:
There is a network name field in openrtb_ext.ExtBid.ExtBidPrebid.Meta.NetworkName that you can set for network name. You won't be needing to create separate struct to pass Network name in the bid request.

Reply:
It will force to change the bid response of the Criteo backend to answer with a ext.prebid.meta.networkName whereas the backend is answering with ext.prebid.networkName

Reply:
ok. I see that you are setting `openrtb_ext.ExtBid.ExtBidPrebid.Meta. NetworkName` in getBidMeta func, so, I think is ok. Approving this PR.